LoopKit.xcodeproj/project.pbxproj

```diff
@@ -3324,6 +3366,7 @@
 				GCC_C_LANGUAGE_STANDARD = gnu11;
 				INFOPLIST_FILE = LoopKitUI/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
+				IPHONEOS_DEPLOYMENT_TARGET = 11.1;
```

Comment:
Please remove these overrides. The default `IPHONEOS_DEPLOYMENT_TARGET` variable is set at the project (not target) level. Specifically, on lines 3452 and 3511. (Although LoopKit has yet to be fully updated, it makes more sense to provide project-level settings and only override at the target-level when necessary.)